### ISSUE TITLE
Make product-add-to-subscription-list.php responsive

### ISSUE THREAD

**Issue by thenbrent** [Prospress/woocommerce-subscribe-all-the-things#353]
At the moment, the matching subscriptions table is not responsive. It should be to avoid display like:

* http://pic.pros.pr/c1612416dd10 or
* http://pic.pros.pr/260a26d321a0

This issue stems from the WCS core `my-subscriptions.php` template, and it has been reported in https://github.com/Prospress/woocommerce-subscriptions/issues/2973. It will be addressed in WCS eventually. I'm opening here though too, as it's likely to be a more common issue with SATT given that placement of the matching subscriptions table would vary more than the **My Account > Subscriptions** table.

**Comment by franticpsyx** [comment]
@thenbrent Looking at this in detail, I can see that:

* The `shop_table` and `shop_table_responsive` classes are correctly added to the table.
* Storefront appears to be picking those up and correctly applying mobile styles: https://cl.ly/6aaa3ded4e0e

Generally I think we're in theme territory here. If a theme includes mobile styles for `shop_table` elements, we may make things worse if we define our own rules.

Although it may seem obvious to style `tr` elements using `display: block; clear: both`, we might be in for surprises/conflicts with themes that follow a different approach.

Summing up, I think a theme's CSS should be general enough to style `shop_table` elements in any page, not just in the cart/checkout pages.

If you follow a different approach in WCS, SATT will simply inherit this, but I'd advise against styling these tables in WCS.

Closing.

**Comment by thenbrent** [comment]
Sounds fair to me 👍
